### PR TITLE
Log version after logging configuration is applied

### PR DESCRIPTION
**Pull request by aludvik**
Signed-off-by: Adam Ludvik <ludvik@bitwise.io>

### CODE REVIEW

File: validator/sawtooth_validator/server/cli.py [hunk 1]
```diff
@@ -272,6 +262,16 @@ def main(args=None):
...
                               name="validator")
 
+    try:
+        version = pkg_resources.get_distribution(DISTRIBUTION_NAME).version
+    except pkg_resources.DistributionNotFound:
+        version = 'UNKNOWN'
+    LOGGER.info(
+        '%s (Hyperledger Sawtooth) version %s', DISTRIBUTION_NAME, version)
+
+    if opts.network_auth:
```
Comment (vaporos) [inline validator/sawtooth_validator/server/cli.py]: Is moving these network auth lines intentional?

**Comment by aludvik** [comment]
nope

**Comment by nick-drozd** [comment]
Could you add a commit message describing what this change is for?

**Comment by aludvik** [comment]
@nick-drozd updated

**Pull request merged**